src/runner.py

```diff
@@ -52,6 +52,10 @@ def is_unregistered(self):
             return False
         return self.local and self.local.status == "Running"
 
+    @property
+    def virt_type(self):
+        return self.local.type if self.local else None
```

Comment:
i haven't tried this but can this be 
```suggestion
        return (self.local or self.remote).type
```